content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -45,17 +46,21 @@ including the `--oauth-no-pkce` flag when starting `chronograf`.
 - Flux improvements and additional functionality:
   - Add dashboard template variables to Flux query execution.
     Flux queries include a `v` record with a key value pair for each variable.
-  - Support dashboard template variables defined with Flux.
-  - Add Kapacitor Flux Tasks on the Manage Tasks page (read only).
+  - Support dashboard [template variables defined with Flux](/chronograf/v1.9/guides/dashboard-template-variables/#flux-query).
+  - Add [Kapacitor Flux Tasks](/chronograf/v1.9/guides/advanced-kapacitor/#manage-kapacitor-flux-tasks)
+    on the Manage Tasks page (read only).
```

Comment:
Do we need to tell folks how to enable/disable/delete TICKscript tasks, or is it obvious on the UI?

Reply:
Looks like we say View TICKscript task activity under Manage Kapacitor Flux tasks, to be sure, is that correct?

Reply:
1. It's pretty self explanatory.
2. Nice catch! I'll fix it.